src/structures/Message.js

```diff
@@ -188,6 +185,16 @@ class Message {
     );
   }
 
+  /**
+   * Represents the author of the message as a guild member
+   * Only available if the message comes from a guild where the author is still a member
+   * @type {?GuildMember}
+   * @readonly
+   */
+  get member() {
```

Comment:
This is a breaking change. It must stay a regular property :(

Reply:
Resolved by https://github.com/discordjs/discord.js/pull/3684/commits/598c3c8ed0ec602b34a380402a2d8bd90f3e7694